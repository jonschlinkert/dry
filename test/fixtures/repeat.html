{% extends "repeat" %}
{% block "foo" %}
foo: "repeat.html"
{% endblock %}