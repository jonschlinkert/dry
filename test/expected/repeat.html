
foo: "repeat.html"

foo: "repeat.html"

foo: "repeat.html"

foo: "repeat.html"

foo: "repeat.html"

foo: "repeat.html"
